backend/chunker/chunker.go

```diff
@@ -150,6 +150,7 @@ Normally should contain a ':' and a path, e.g. "myremote:path/to/dir",
 		}, {
 			Name:     "name_format",
 			Advanced: true,
+			Hide:     fs.OptionHideCommandLine,
```

Comment:
You can `HideBoth` if you want `rclone config` to no longer offer them to users. They will carry on being read from existing config files.

Reply:
Yet I would like to have it offered for new remotes.